src/components/Review/Review.tsx

```diff
@@ -17,233 +17,364 @@ import { CustomToggleAccordion } from "../CustomToggleAccordion/CustomToggleAcco
+      setModalActivityId(-1);
+    } catch (e) {
+      console.error(e);
+      setError(true);
+      setLoading(false);
     }
+  };
 
-    const cardOnClick = (activity: any) => {
-        setModalActivityId(activity.Id);
+  const deleteActivity = async (activity: any) => {
+    try {
+      setDeleting(true);
+      await activitiesApi.deleteActivity(
+        await ActivityUtilities.buildActivity(activity)
+      );
+      setActivities(
+        groupActivities(
+          ActivityUtilities.filterActivity(getAllActivities(), activity)
+        )
+      );
+      setDeleting(false);
+      setModalActivityId(-1);
+    } catch (e) {
+      console.error(e);
+      setError(true);
+      setDeleting(false);
+      setModalActivityId(-1);
     }
+  };
 
-    const isOrgDisplayed = (org: string) => {
-        return !defaultOrg || defaultOrg === org || (defaultIncludeSubOrgs && org.includes(defaultOrg));
-    }
+  const closeModal = () => {
+    setModalActivityId(-1);
+  };
 
-    useEffect(() => {
-        fetchActivities();
-        // eslint-disable-next-line
-    }, [urlQuery, urlOrg, urlIncludeSubOrgs, urlStartDate, urlEndDate, urlIsHistory, urlIsMAR, urlOpr]);
+  const cardOnClick = (activity: any) => {
+    setModalActivityId(activity.Id);
+  };
 
+  const isOrgDisplayed = (org: string) => {
     return (
-        <Container fluid>
-            <Row className="justify-content-center m-3"><h1>{defaultQuery === null ? "Review Activities" : "Search Results"}</h1></Row>
-            <CardAccordion defaultOpen={false} cardHeader="Search and Filter">
-                <SearchForm
-                    defaultQuery={defaultQuery}
-                    defaultOrg={defaultOrg}
-                    defaultIncludeSubOrgs={defaultIncludeSubOrgs}
-                    defaultStartDate={defaultStartDate ? DateUtilities.getDate(defaultStartDate) : undefined}
-                    defaultEndDate={defaultEndDate ? DateUtilities.getDate(defaultEndDate) : undefined}
-                    defaultIsHistory={defaultIsHistory}
-                    defaultIsMAR={defaultIsMAR}
-                    defaultOpr={defaultOpr ? defaultOpr : null}
-                    loading={loading}
-                    orgs={orgs ? orgs : []}
-                />
-            </CardAccordion>
-            {activities.map(activitiesForWeek =>
+      !defaultOrg ||
+      defaultOrg === org ||
+      (defaultIncludeSubOrgs && org.includes(defaultOrg))
+    );
+  };
+
+  useEffect(() => {
+    fetchActivities();
+    // eslint-disable-next-line
+  }, [
+    urlQuery,
+    urlOrg,
+    urlIncludeSubOrgs,
+    urlStartDate,
+    urlEndDate,
+    urlIsHistory,
+    urlIsMAR,
+    urlOpr,
+  ]);
+
+  return (
+    <Container fluid>
+      <Row className="justify-content-center m-3">
+        <h1>
+          {defaultQuery === null ? "Review Activities" : "Search Results"}
+        </h1>
+      </Row>
+      <CardAccordion defaultOpen={false} cardHeader="Search and Filter">
```

Comment:
You don't pass any props here, so the expansion triangle doesn't keep track of expanded/collapsed, and always shows icon for "expanded" even if really collapsed.